src/node.h

```diff
@@ -391,7 +391,8 @@ inline void NODE_SET_PROTOTYPE_METHOD(v8::Local<v8::FunctionTemplate> recv,
 #define NODE_SET_PROTOTYPE_METHOD node::NODE_SET_PROTOTYPE_METHOD
 
 // BINARY is a deprecated alias of LATIN1.
-enum encoding {ASCII, UTF8, BASE64, UCS2, BINARY, HEX, BUFFER, LATIN1 = BINARY};
+enum encoding {ASCII, UTF8, BASE64, UCS2,
+               BINARY, HEX, BUFFER, LATIN1 = BINARY, PARSE_ERROR};
```

Comment:
This is a public API. Avoid extending it for an internal use case. Same goes for ParseEncoding().